lib/scrabble.rb

```diff
@@ -10,4 +13,36 @@ def point_values
       "Y"=>4, "Z"=>10
     }
   end
+
+  def score(word)
```

Comment:
This looks great - I think there is some room for refactoring as you mentioned, but really great job!